### DISCUSSION TITLE
datepicker how to used when combine with mysql added query

### DISCUSSION THREAD

**Discussion by Pierrelai99** [Budibase/budibase#4499] [Help]
![image](https://user-images.githubusercontent.com/99549872/154026112-b019c065-79cf-4dfc-b856-81573f822b5d.png)

my database is mysql,
I have add the datepicker under the data provider like below image, but not sure why the datepicker didn't have fields to choose show at top photo, can somebody help me pls, thanks a lot
![image](https://user-images.githubusercontent.com/99549872/154026308-3d0e8537-127f-4c42-a3f5-3099c294d383.png)

**Comment by mjashanks** [comment]
Hey @Pierrelai99 - you are missing a `Releater`. Your structure should look like:

```
> Data Provider
  > Repeater
     > Container
     etc....
```
